cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/store/AppMetadataStore.java

```diff
@@ -34,7 +34,8 @@
  * JIRA https://issues.cask.co/browse/CDAP-2172
  */
 public class AppMetadataStore extends MetadataStoreDataset {
-  public static final String TYPE_RUN_RECORD_STARTED = "runRecordStarted";
+  public static final String TYPE_RUN_RECORD_STARTED = "runRecordStarting";
```

Comment:
keep naming consistent as in `cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/AppMetadataStore.java`